src/Workspaces/CSharp/Portable/FindSymbols/CSharpDeclaredSymbolInfoFactoryService.cs

```diff
@@ -147,7 +154,16 @@ private static void ProcessUsings(List<Dictionary<string, string>> aliasMaps, Sy
             }
         }
 
-        public override bool TryGetDeclaredSymbolInfo(StringTable stringTable, SyntaxNode node, string rootNamespace, out DeclaredSymbolInfo declaredSymbolInfo)
+        protected override void AddDeclaredSymbolInfosWorker(
+            SyntaxNode container,
```

Comment:
by passing in the container, we save a lot of ugly computation in the VB side trying to figure out waht the container is for any particular member/type.